.travis.yml

```diff
@@ -38,6 +38,33 @@ matrix:
                 - clang-format-5.0
           script:
             - ./scripts/run-clang-format.py -r src
+        - os: osx
+          osx_image: xcode9.3
```

Comment:
What's the difference between 9.3 and 9.2? I suspect there is some, so can you please document it?